griptape/events/start_structure_run_event.py

```diff
@@ -9,6 +9,7 @@
 
 @define
 class StartStructureRunEvent(BaseEvent):
```

Comment:
Right now our template illustrates how to send a FINISH event, but should we also train them on START events? Or are these automatic by the service? Is there any value for a Structure-that-doesn't-use-Structures to send a Start event?

Reply:
The `FinishStructureRunEvent` is all that is needed for the runtime. User's publishing a `StartStructureRunEvent` would have no effect today.

Reply:
But this _does_ remind me that we need to update the manual publish example to include this new required field `structure_id`.